python/py-sentence-transformers/Portfile

```diff
@@ -60,10 +64,11 @@ if {${name} ne ${subport}} {
         set sharedir ${prefix}/share/${subport}
         xinstall -d \
             ${destroot}${docdir}
-        xinstall -m 0644 -W ${worksrcpath} LICENSE NOTICE.txt README.md \
+        xinstall -m 0644 -W ${worksrcpath} README.md \
             ${destroot}${docdir}
-        copy ${worksrcpath}/docs/pretrained-models ${destroot}${docdir}
-        copy ${worksrcpath}/examples ${destroot}${sharedir}
+        # this documentation exists in the GitHub Repo
+        # copy ${worksrcpath}/docs/pretrained-models ${destroot}${docdir}
+        # copy ${worksrcpath}/examples ${destroot}${sharedir}
```

Comment:
do you intend to switch back to GitHub next time if/when upstream tags a release? If so, then you can leave the lines above; otherwise please remove.

Reply:
If the authors keep most recent release with fixes on pypi, then let's keep that source.